source/default/_patterns/02-molecules/tabs/__tests__/tabs.test.js

```diff
@@ -0,0 +1,144 @@
+import $ from 'jquery';
+import tabs, { name, selectedIndexState, updateSelectedState } from '..';
+
+const markup = `
```

Comment:
Is this necessarily the pattern we want to use for testing markup generated by our patterns? Since test coverage of default patterns is minimal, this will become the defacto procedure for developers to follow. Would it be possible to capture the generated content from Pattern Lab and test against that markup? If so, does it then make more sense to move our tests into demos?